framework/packages/abstract-interface/src/deployers.rs

```diff
@@ -17,6 +20,10 @@ pub trait RegisteredModule {
     fn module_id<'a>() -> &'a str;
     /// The version of the module.
     fn module_version<'a>() -> &'a str;
+    /// The id of the module for the account
+    fn account_module_id(account_id: &AccountId) -> String {
```

Comment:
Don't really like this function name... What do you think of `installed_module_contract_id`?

Problem is that we're using "id" in so many places. So we need to be increasingly specific about what kind of ID we're talking about. 

Think "contract_id" makes it clear that it's used for cw-orch.